### PR TITLE
Add FC data link to Saturn Data Library

### LINKED ISSUE #768: Link to the FireCloud data library from Saturn Library
Users need a way in Saturn to find the following categories of dataset:
1) Datasets sequenced at Broad and delivered through FireCloud / Saturn
2) Public datasets hosted at Broad, such as TCGA, TARGET, etc.

Both of these classes of dataset are hosted FC/Saturn as workspaces, catalogued via a rich dataset metadata schema in the FireCloud data library.
https://portal.firecloud.org/#library

Until such time as these data are discoverable by a dedicated object-level explorer, we will surface them in the Saturn library by adding a new entry:
* Name: FireCloud Dataset Library
* Image: Broad logo
* Participants: XXXX
* Description: "Search for datasets sequenced at the Broad Institute, or public datasets hosted at the Broad. Datasets are pre-loaded as workspaces. You can clone these, or copy data into the workspace of your choice"

### PR DESCRIPTION
**Pull request by katevoss**
Fixes #768

### CODE REVIEW

**Review by elizzhao — APPROVED** [review]
Looks good to me!

**Review by panentheos — APPROVED** [review]
Looks good.

File: src/pages/BrowseData.js [hunk 1]
```diff
@@ -201,6 +202,21 @@ const gtex = h(Participant, {
...
+const fcDataLib = h(Participant, {
+  logo: { src: broadLogo, alt: 'Broad logo', height: '40%' },
+  title: 'FireCloud Dataset Library',
+  description: `Search for datasets sequenced at the Broad Institute, or public datasets hosted at the Broad. Datasets
+   are pre-loaded as workspaces. You can clone these, or copy data into the workspace of your choice.`,
+  sizeText: h(TooltipTrigger, { content: 'As of October 2018' }, [span('Samples: > 158,629')])
+}, [
+  buttonPrimary({
+    as: 'a',
+    href: 'https://portal.firecloud.org/#library',
```
Comment (MattPutnam) [inline src/pages/BrowseData.js]: Should we make this point to dev for dev?

**Review by MattPutnam — APPROVED** [review]

File: src/pages/BrowseData.js (hunk 1 above)
Comment (katevoss) [reply src/pages/BrowseData.js]: How would I make dev point to dev, prod to prod, etc? Would I use the `firecloudUrlRoot` from the config?

**Pull request merged**